test/cli/fixtures/unhandled-rejection.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+QUnit.module( "Unhandled Rejections", function() {
+	QUnit.test( "test passes just fine, but has a rejected promise", function( assert ) {
+		assert.ok( true );
+
+		const done = assert.async();
+
+		new Promise( function( resolve ) {
```

Comment:
Correct me if I'm wrong, but can't this just be `Promise.resolve().then( ... )`? If so, can you change it here and below as well.

Reply:
Yep, you are absolutely correct. I have been working too much with Ember's internal `Ember.RSVP.Promise` which is configured to resolve in an "odd" way, sorry about that!

Updated to be much more idiomatic...